docs/security.md

```diff
@@ -947,7 +947,7 @@ mechanism (see `java.util.ServiceLoader`). Implementations of
 `org.apache.spark.security.HadoopDelegationTokenProvider` can be made available to Spark
 by listing their names in the corresponding file in the jar's `META-INF/services` directory.
 
-Delegation token support is currently only supported in YARN mode. Consult the
+Delegation token support is currently only supported in YARN and Kubernetes mode. Consult the
 deployment-specific page for more information.
 
 The following options provides finer-grained control for this feature:
```

Comment:
There is another place:

```
Users can exclude Kerberos delegation token renewal at resource scheduler. Currently it is only supported
on YARN.
```

Reply:
> Consult the deployment-specific page for more information.

Btw, for YARN, it is true that we have a corresponding section for it as following. Do we have same one for Kubernetes? If so, can you also add the link?

```
The configuration is covered in the [Running Spark on YARN](running-on-yarn.html#yarn-specific-kerberos-configuration) page.
```

Reply:
@viirya DT renewal exclusion is another feature that is only supported by YARN.

In YARN mode, the RM could take responsibility for the token renewal, in K8s

>  there isn't a definition of what "external renewal service" is anywhere

ref SPARK-25825

Reply:
> Btw, for YARN, it is true that we have a corresponding section for it as following. Do we have same one for Kubernetes?

Unlike YARN, the introduction of the K8s Kerberos support was embedded on the security page.

Reply:
> Unlike YARN, the introduction of the K8s Kerberos support was embedded on the security page.

Do you mean the section "Secure Interaction with Kubernetes"?

Reply:
Yes.